pkg/util/error.go

```diff
@@ -40,3 +40,16 @@ func CheckErr(err error, handleErr func(string)) {
 	}
 	handleErr(err.Error())
 }
+
+// CheckErrDefault to handle command errors with default handler
+func CheckErrDefault(err error, msg string) {
```

Comment:
This function is not super required IMO, for golang, devs usually prefer to handle the error in the same place where they are generated, this might seem a bit counter-intuitive at first.

Ref: https://go.dev/blog/error-handling-and-go
Ref: https://go.dev/blog/errors-are-values

Reply:
Removed, Thanks.